app/src/main/java/com/example/pokedexapp/data/network/PokemonSpeciesApiDto.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.pokedexapp.data.network
+
+data class PokemonSpeciesApiDto(
+    val evolves_from_species: PokemonListItemApiDto?
```

Comment:
Por que o underscore ( _ ) ? Se for por causa da nomemclatura na api,  acho que tem um @ do json que pode colocar em cima do atributo pra dizer qual é o nome na api e poder usar um evolvesFromSpecies aqui